package.json

```
@@ -0,0 +1,40 @@
+{
+  "name": "crudp",
+  "version": "1.0.0",
+  "description": "This  project shows the crud operation Focus area to create this ui design in typescript",
```

Minor nit: Please be consistent with naming convention.
```suggestion
  "description": "This  project shows the crud operation focus area to create this UI design in typescript",
```

---

change name https://github.com/Sakshi-Simform/product-management/pull/1/commits/49da5cce6523e5a88134bd522bfac2cfdb778a9d